### DISCUSSION TITLE
FYI I have something less complete but with other features

### DISCUSSION THREAD

**Discussion by max-sixty** [PatrickF1/fzf.fish#249] [Ideas]
Hi @PatrickF1 — great work on this library!

FYI I've been gradually building something similar focused on git. It's much less complete than yours, but has some cool features that might be interesting to integrate.

Specifically, with a `C-g C-` prefix, we get:
  - `b` branches, `t` tags, `s` status, `l` log
  - All with `fzf`'s fuzzy search, scrollable previews (e.g. log has a preview of the commit, branch has a preview of the log)
  - Some with more actions — e.g. `C-s` switches to a branch, `C-a` adds a file

It's at https://gist.github.com/max-sixty/3ce6b262b650ac5b94f94c9c190ef7d4 if there are any ideas that you think are worth taking.

**Comment by PatrickF1** [comment]
Thank you Max! Very interesting, nice work. I'm pretty content with my features though but I hope fzf.fish users who want more will take a look at your gist and use them as well
